### PR TITLE
feat: mobile toolbar basic UI

### PR DESCRIPTION
**Pull request by hyj1204**
1. Import all the toolbar icons' SVG files
2. Create `AFMobileIcon` to show the icons used on mobile. `AFMobileIcons` is an enhanced enum to prevent name misspellings.  Usage is similar to the `Icon` widget.

image

3. Currently, I just simply placed all the icons as Iconbuttons on the toolbar. The features will implement in the future PRs.

https://github.com/LucasXu0/appflowy-editor/assets/14248245/8261b2c4-2b9b-4c99-b175-0f9f0ce4a22c

### CODE REVIEW

**Comment by LucasXu0** [comment]
@hyj1204 Is it ready for merging?

File: lib/src/editor/toolbar/mobile_toolbar.dart [hunk 1]
```diff
@@ -11,6 +12,23 @@ class MobileToolbar extends StatelessWidget {
...
+      ),
+      // TODO(yijing): Implement features later
+      ...List.generate(
+          16,
+          (index) => IconButton(
+              onPressed: () {},
+              icon: AFMobileIcon(
+                afMobileIcons: AFMobileIcons.values.toList()[index],
+              ))),
+    ];
```
Comment (LucasXu0) [inline lib/src/editor/toolbar/mobile_toolbar.dart]: the `toolbarItems` should be a parameter.

```dart
  const MobileToolbar({
    super.key,
    required this.editorState,
    required this.toolbarItemWidgets,
  });

  final EditorState editorState;
  final List<Widget> toolbarItemWidgets;
```

File: lib/src/editor/toolbar/mobile_toolbar.dart [hunk 2]
```diff
@@ -11,6 +12,23 @@ class MobileToolbar extends StatelessWidget {
...
+        icon: const Icon(Icons.keyboard_hide),
+      ),
+      // TODO(yijing): Implement features later
+      ...List.generate(
+          16,
+          (index) => IconButton(
+              onPressed: () {},
+              icon: AFMobileIcon(
+                afMobileIcons: AFMobileIcons.values.toList()[index],
+              ))),
```
Comment (LucasXu0) [inline lib/src/editor/toolbar/mobile_toolbar.dart]: add trailing comma.

Comment (LucasXu0) [reply lib/src/editor/toolbar/mobile_toolbar.dart]: use map to generate the list would be better.

```dart
      ...AFMobileIcons.values
          .map(
            (e) => IconButton(
              onPressed: () {},
              icon: AFMobileIcon(
                afMobileIcons: e,
              ),
            ),
          )
          .toList(),
```

**Comment by LucasXu0** [comment]
Just submit the PR to the appflowy-editor repo. This branch will be deprecated after the PR is merged. https://github.com/AppFlowy-IO/appflowy-editor/pull/129

**Comment by hyj1204** [comment]
@LucasXu0 
Got it and it still needs to be pushed to the `mobile` branch right?

**Pull request merged**